be/tool/main.c

```diff
@@ -32,12 +32,27 @@
 
 #include <stdio.h>              /* printf */
 #include <stdlib.h>             /* EXIT_FAILURE */
+#include <string.h>     	/* strcmp */
 
 #include "lib/string.h"         /* m0_streq */
+#include "lib/errno.h"          /* ENOMEM */
 
 #include "be/tool/st.h"         /* m0_betool_st_mkfs */
 #include "be/tool/common.h"     /* m0_betool_m0_init */
 #include "be/ut/helper.h"       /* m0_be_ut_backend */
+#include "be/alloc_internal.h"  /* m0_be_allocator_header */
+#include "stob/ad_private.h"    /* stob_ad_0type_rec */
+#include "cob/cob.h"            /* m0_cob_domain */
+#include "stob/ad.h"            /* m0_stob_ad_domain */
+#include "be/extmap_internal.h" /* m0_be_emap */
+#include "balloc/balloc.h"      /* m0_balloc */
+#include "be/domain.h"	       /*m0_be_segobj_opt_begine,m0_be_segobj_opt_next*/
+#include "lib/uuid.h"		/* m0_node_uuid_string_set */
```

Comment:
These two are the same as previous one.

Use spaces in stead of tab in the middle of a line.